### PR TITLE
merge sharing screens

### PR DESCRIPTION
**Pull request by kevodwyer**
peergos PR: https://github.com/Peergos/Peergos/pull/693

### CODE REVIEW

**Comment by ianopolous** [comment]
Can you post a screenshot what the new modal looks like?

File: src/components/filesystem/index.js [hunk 1]
```diff
@@ -317,21 +316,21 @@ module.exports = {
 		this.showHexViewer = true;
 	},
 
-	updateCurrentDir: function() {
+	updateCurrentDir: function(refreshSharing) {
```
Comment (ianopolous) [inline src/components/filesystem/index.js:319]: I'm a little nervous about coupling the sharing update to the dir directly like this. There are also other calls to this method that are not passing an arg now.

**Comment by kevodwyer** [comment]
> Can you post a screenshot what the new modal looks like?

sharescreen

File: src/components/filesystem/index.js (hunk 1 above)
Comment (kevodwyer) [reply src/components/filesystem/index.js:319]: It is only relevant in the one case when forceSharedWithUpdate is triggered in order to refresh the sharedWithData data structure.

**Pull request closed without merging**